### PR TITLE
Feat/scout 201 matches tabs

### PR DESCRIPTION
**Pull request by dejwi**
### Task Description

[SCOUT-201](https://playmakerpro.atlassian.net/browse/SCOUT-201?atlOrigin=eyJpIjoiZWE4N2U4OGNiZDQwNDc5NGE4NWExMzIxZWIxNmJhNGUiLCJwIjoiaiJ9)

- added tab for each team at match page with notes&reports
- changed some componenets

something like this ?

### CODE REVIEW

**Comment by jakubfiglak** [comment]
> ### Task Description
> [SCOUT-201](https://playmakerpro.atlassian.net/browse/SCOUT-201?atlOrigin=eyJpIjoiZWE4N2U4OGNiZDQwNDc5NGE4NWExMzIxZWIxNmJhNGUiLCJwIjoiaiJ9)
> 
> * added tab for each team at match page with notes&reports
> * changed some componenets
> 
> something like this ?

I'd try and use different variant of tabs so that it clearly contrasts with the background - something like this perhaps https://mui.com/material-ui/react-tabs/#full-width

Also, I'd create 4 tabs and include `total` documents count in the tab title, like so:
- team1 - notes (8)
- team1 - reports (0)
- team2 - notes (6)
- team2 - reports (2)

**Comment by dejwi** [comment]
What about something like this? We can't have team name in every tab because it keeps one line format and on small(mobile) screen it looks like someone puked there

File: src/pages/matches/[id].tsx [hunk 1]
```diff
@@ -21,21 +36,198 @@ export const getServerSideProps = withSessionSsrRole<MatchDto>(
...
+    </>
+  )
 }
 
+// const SectionHeading = ({ title }: { title: string }) => (
+//   <Typography variant="h3" align="center" paddingY={theme => theme.spacing(2)}>
+//     {title}
+//   </Typography>
+// )
+
```
Comment (jakubfiglak) [inline src/pages/matches/[id].tsx:232]: Please remove commented code

**Review by jakubfiglak — APPROVED** [review]
Nice, I like it! Please just remove commented code and we're good to go

**Pull request merged**